COMMON_ISSUES.md

```diff
@@ -28,6 +28,11 @@ You may also experience codec issues in Peer-to-Peer (P2P) Rooms containing
 Android- or iOS-based Participants who do not support H.264. However, P2P Rooms
 with browser-based Participants should work.
 
+Also, there is an existing bug in twilio-video.js where publication of
+DataTracks after joining a group Room never reaches completion. While we work
+to fix this bug, you can work around this by publishing your DataTracks
+while connecting to a Room using the `tracks` property of ConnectOptions.
```

Comment:
Can you please include the JSDK issue number here? Also, is it a bug in twilio-video.js or the media server or Safari?

Reply:
@markandrus Although this might be a media server error, Firefox and Safari are the browsers that are affected. +1 on mentioning the ticket.